### PR TITLE
Update README and deps

### PR DESCRIPTION
**Pull request by marctemp**
https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/121769

### CODE REVIEW

File: helm/ffc-pay-enrichment/values.yaml [hunk 1]
```diff
@@ -19,17 +19,12 @@ containerSecret:
   type: Opaque
 
 deployment:
```
Comment (marctemp) [inline helm/ffc-pay-enrichment/values.yaml:21]: Can see why these are not required via here: https://github.com/DEFRA/ffc-helm-library/blob/master/ffc-helm-library/templates/_deployment.yaml

File: helm/ffc-pay-enrichment/values.yaml [hunk 2]
```diff
@@ -19,17 +19,12 @@ containerSecret:
...
-  restartPolicy: Always
-  priorityClassName: default
-  replicas: 1
   minReplicas: 1
   maxReplicas: 16
   cpuThreshold: 80
   memoryThreshold: 80
-  minReadySeconds: 0
 
 container:
```
Comment (marctemp) [inline helm/ffc-pay-enrichment/values.yaml:27]: And here: https://github.com/DEFRA/ffc-helm-library/blob/master/ffc-helm-library/templates/_container.yaml

**Comment by AbidemiAdio** [comment]
I can find contributing section or related contributing.md

**Comment by marctemp** [comment]
Was something we didn't feel was very valuable and decided to not include it in the proposed structure: https://eaflood.atlassian.net/wiki/spaces/SFI/pages/3990979123/README+Structure#Proposed-README-Structure

Do you think it's worth adding?

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**